CARDS-2015 - Patient portal: if there's only one questionnaire in the survey, don't display the step indicator (1/1)

marta- opened the pull request.

marta- committed:
CARDS-2015 - Patient portal: if there's only one questionnaire in the survey, don't display the step indicator (1/1)

sashaandjic reviewed (approved):
tested

One survey:
![image](https://user-images.githubusercontent.com/23243244/213362001-eb9c6d8e-82ed-4dac-8a81-3b42414f0fd4.png)

Two surveys:
![image](https://user-images.githubusercontent.com/23243244/213362052-87d4868e-044b-484b-9f53-22dbe31c6dd2.png)

sdumitriu commented on modules/patient-portal/src/main/frontend/src/patient-portal/QuestionnaireSet.jsx:
```diff
@@ -390,7 +390,7 @@ function QuestionnaireSet(props) {
   let stepIndicator = (step, withTotal) => {
     return (
-      step >=0 && step < questionnaireIds?.length ?
+      step >=0 && (!withTotal || questionnaireIds?.length > 1) && step < questionnaireIds?.length ?
```
`!withTotal` is here to make sure that we always show the number next to the questionnaire title in the summary screen, but should we do that? Isn't it better to not show a number next to the only thing to do?

veronikaslc replied:
+1, probably we can remove `!withTotal`

marta- replied:
Without any other changes, simply removing the (1) on the start page ruins the overall look as [the survey name becomes less noticeable](https://user-images.githubusercontent.com/651980/213588469-0bcaa56f-a067-4e00-9636-382d413b4047.png). I've [replaced (1) with an icon](https://user-images.githubusercontent.com/651980/213589596-c02109f1-7e81-4d76-ac90-96371b94853b.png) on the start page in commit c71334e.

marta- committed:
CARDS-2015 - Patient portal: if there's only one questionnaire in the survey, don't display the step indicator (1/1)
Also replace (1) with an icon on the start page if there's only one form to be filled out

sashaandjic reviewed (commented):
The latest change was tested in dev post merge: if there is only one survey to fill in show an icon instead of `1`

![image](https://user-images.githubusercontent.com/23243244/213614551-c7055ca8-06e0-480f-be87-7de388e85941.png)
![image](https://user-images.githubusercontent.com/23243244/213614831-b320be3b-8712-4a2e-ae2b-d8cf41042e1e.png)
